src/common.h

```diff
@@ -92,7 +92,7 @@
 #define		SF_MIN(a, b)	((a) < (b) ? (a) : (b))
 
 
-#define		COMPILE_TIME_ASSERT(e)	(sizeof (struct { int : - !! (e) ; }))
+#define		COMPILE_TIME_ASSERT(e)	(sizeof (struct { int : - ! (e) ; }))
```

Comment:
Was this change inclusion intentional? Would seem to reverse the logic of the check.

Reply:
It's not used anywhere else, and reversing the check means that it more closely matches the behaviour of `assert()`.